moped-editor/src/views/projects/projectView/ProjectActivityEntry.js

```diff
@@ -2,51 +2,56 @@ import React from "react";
 import { Box, Typography, Link } from "@mui/material";
 import { NavLink as RouterLink } from "react-router-dom";
 
-import makeStyles from "@mui/styles/makeStyles";
-
-const useStyles = makeStyles(() => ({
-  entryText: {
-    padding: "0 0 0 .5rem",
-  },
-  boldText: {
-    fontWeight: 600,
-  },
-  indentText: {
-    paddingLeft: "16px",
-    display: "block",
-  },
-}));
-
 /**
  * One row/entry in the project activity log
  * @param {changeIcon} Material UI icon component
  * @param {changeText} - Array of objects of the shape:
  *  {text: String to display, style: String name of style or null}
  */
 const ProjectActivityEntry = ({ changeIcon, changeText }) => {
-  const classes = useStyles();
+  const getStyleSx = (style) => {
```

Comment:
@mateoclarke could you move this function outside this component body? I'm only mentioning since some projects have a bunch of activity entries that would cause it to be created that many times.